ivadomed/losses.py

```diff
@@ -252,3 +254,102 @@ def forward(self, input, target):
 
         # TODO: Take the opposite?
         return focal_tversky_sum / n_classes
+
+
+class L2loss(nn.Module):
+    """
+    L2 loss between two images : inputs and target
+    """
+
+    def __init__(self):
+        super(L2loss, self).__init__()
+
+    def forward(self, input, target):
+        return torch.sum((input - target) ** 2) / 2
+
+
+class AdapWingLoss(nn.Module):
+    """
+    adaptive Wing loss : https://arxiv.org/abs/1904.07399
+    Used for heatmap ground truth.
+
+    """
+
+    def __init__(self):
+        super(AdapWingLoss, self).__init__()
+
+    def forward(self,input,target):
+        theta = 0.5
```

Comment:
Do we want to put these as parameters that we could change (theta, alpha, w, e)? Also for clarity, if it's possible, can you use more explicit variable names for w, e, A and C?

Reply:
Yep will do !